app/helpers/utils.rb

```diff
@@ -27,4 +27,10 @@ def self.mail_notice(input_mails, parsed_mails, newly_added)
                "No valid Email(s) entered."
              end
   end
+
+  # To strip tags from input data
+  # @param input data string
+  def self.stripTags(input)
+    input.present? ? input.gsub(/(<([^>]+)>)/iu, "").strip() : nil
+  end
```

Comment:
Is there a built-in rails function to do this?

Reply:
Yes, as tachyons also pointed out, using sanitize helper now.